release/build.xml

```diff
@@ -43,23 +43,19 @@
     <os family="unix"/>
   </condition>
 
-  <!-- Project build properties -->
-  <property name="appName" value="GeoNetwork"/>
+  <!-- Import global build properties -->
+  <property file="./build.properties"/>
+
+  <!-- Additional build properties -->
   <property name="bundleName" value="geonetwork-bundle"/>
-  <property name="displayName" value="GeoNetwork opensource"/>
-  <property name="homepage" value="https://geonetwork-opensource.org"/>
-  <property name="version" value="3.11.0"/>
-  <property name="subVersion" value="SNAPSHOT"/>
-  <property name="javaVersion" value="1.8.0"/>
-  <property name="jreUrl" value="http://openjdk.java.net/"/>
-  <property name="OS" value="Compiled on ${os.name} (${osys})"/>
```

Comment:
Note that the `OS` property is a remnant of the old JAR installer and is no longer required.